x-pack/plugins/maps/public/store/store.js

```diff
@@ -20,17 +18,12 @@ const enhancers = [ applyMiddleware(thunk) ];
 window.__REDUX_DEVTOOLS_EXTENSION
   && enhancers.push(window.__REDUX_DEVTOOLS_EXTENSION__());
 
-export const getStore = function () {
-  if (store) {
-    return store;
-  }
-
+export function createMapStore() {
   const storeConfig = {};
   const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Is this dev-tools inclusion going to hurt this at all? I never use this debug tool, but wondering how it would behave when multiple stores are on the page.

Anybody using it? 

We probably should not initialize this either in prod. Maybe rename `showMapsInspectorAdapter` and have that flag govern this as well. cc @aaronjcaldwell 

fwiw: for me I got it to crash the page when trying to use the extension and use the playback-feature:

![image](https://user-images.githubusercontent.com/1833023/52745132-5a74df80-2fac-11e9-875c-bf77fb012746.png)

That might also be PEBKAC

Reply:
I am plus one for removing the debug tools

Reply:
> Is this dev-tools inclusion going to hurt this at all? I never use this debug tool, but wondering how it would behave when multiple stores are on the page

Regardless of our implementation, there will be multiple stores on the page. Dashboard has its own redux store and other embeddables could each have their own redux store.

Reply:
This is being used in several places, do a search for `__REDUX_DEVTOOLS_EXTENSION` and you'll find a few examples (i.e.- apm, cross-cluster replication, etc.). I would vote not to remove this. It's actually really valuable for debugging in dev or prod and only really activates if the browser has the extension installed. I actually use it heavily.

Reply:
If its being used when we can keep it. Will having multiple store instances effect it?

Reply:
That shouldn't be an issue as there's only one instance ever active at a time, passed into the root of the react app in the map controller here:
```
<Provider store={store}>
        <I18nContext>
          <GisMap/>
        </I18nContext>
      </Provider>
```
I was able to verify this locally

Reply:
But once embedded, each embeddable will have this 
```
<Provider store={store}>
        <I18nContext>
          <GisMap/>
        </I18nContext>
      </Provider>
```

Reply:
Mhm, good point. I believe this should be covered though since version 2.0: https://github.com/zalmoxisus/redux-devtools-extension/releases/tag/v2.0.0. It looks like we could pass an instanceId per store but it's not required.

Reply:
Update, the plugin actually did crash eventually and wouldn't recover without restarting chrome. The good news is, it had no effect on the app.

Reply:
Let's go ahead and remove the dev tools for now. As a separate effort, I may attempt to add back in as a separate PR if I can prove stability.